lib/swift/Tests/ThriftTests/TJSONProtocolTests.swift

```diff
@@ -0,0 +1,212 @@
+//
+//  File.swift
+//
+//  Created by Khodabakhsh, Hojjat on 5/18/23.
+//
+
+import XCTest
```

Comment:
please replace the file header by std ASF license header

Reply:
will do.